### ISSUE TITLE
The window is not properly rendered, Mint 18

### ISSUE THREAD

**Issue by surenenfiajyan** [linuxmint/xplayer#32]
When I open Xplayer, the window contents are often not properly rendered, also can happen during resizing. Here are some screenshots

![videos1](https://cloud.githubusercontent.com/assets/13224708/17273003/831864fa-56b7-11e6-8a7b-642b7ab797e0.png)
![videos2](https://cloud.githubusercontent.com/assets/13224708/17273004/831c1a46-56b7-11e6-8d9c-6ffdc21ab8f5.png)

**Comment by Vahan86** [comment]
I have the same issue.

![untitled](https://cloud.githubusercontent.com/assets/12819058/18800699/71977514-81ef-11e6-8c57-e0353fe63527.png)

**Comment by marmistrz** [comment]
Same here 
![screenshot from 2016-09-26 11-16-28](https://cloud.githubusercontent.com/assets/2914938/18829226/a040dce6-83db-11e6-9c40-f5ee188d7fb2.png)

**Comment by JosephMcc** [comment]
I don't use xplayer a bunch but I do use it and haven't seen anything like that.

**Comment by surenenfiajyan** [comment]
@Vahan86  @marmistrz  Does this happen in MATE (Mint 18) ? I couldn't reproduce this in MATE.

**Comment by marmistrz** [comment]
@surenenfiajyan Not using either MATE or Mint.

**Comment by Vahan86** [comment]
@surenenfiajyan sorry, but I am not using Mate.

**Comment by Vahan86** [comment]
@JosephMcc, in most cases the issue occurs during resizing the xplayer window.

**Comment by marmistrz** [comment]
Indeed, it happens while resizing the window, there are sweets spots in which the display is not corrupt.

**Comment by JosephMcc** [comment]
Could you guys please post the Os, desktop environment, video card, and driver you are using. See what is common among all of you having the issue.

**Comment by surenenfiajyan** [comment]
@JosephMcc Mint 18 Cinnamon x86_64 intel i3 2120 (intel HD 2000 GPU). I don't remember the driver, I guess it was the default one for this GPU.

**Comment by marmistrz** [comment]
@JosephMcc Arch Linux, Cinnamon 3.0.7, Intel i5-5200U (Intel HD Graphics 5500), driver `xf86-video-intel`

**Comment by marmistrz** [comment]
The bug is gone when I tried under Plasma. This suggests Cinnamon is the culprit.

**Comment by Vahan86** [comment]
Linux Mint 18 Cinnamon 64-bit, Intel i7-4710HQ, Intel Corporation 4th Gen Core Processor Integrated Graphics Controller.

**Comment by JosephMcc** [comment]
It sort of looks like the Cinnamon/Intel combination might be the problem. Could be why I haven't seen it since I'm using Nvidia.

**Comment by J0Ar** [comment]
I'm on Linux Mint 18 Xfce and i also have this Problem. Sometimes on start the full window has glitches and always by resizing.

>  **Kernel**: 4.4.0-45-generic x86_64 (64 bit) 
>  **Desktop**: Xfce 4.12.3  **Distro**: Linux Mint 18 Sarah
>  **CPU**: Dual core Intel Core i3-3225 (-HT-MCP-)
>  **Graphics Card**: Intel Xeon E3-1200 v2/3rd Gen Core processor Graphics Controller
>  **Display Server**: X.Org 1.18.4  **Driver**: intel  **Resolution**: 1920x1080@60.00hz
>  **GLX Renderer**: Mesa DRI Intel Ivybridge Desktop **GLX Version**: 3.0 Mesa 11.2.0

**Comment by Zac610** [comment]
It is indeed a Intel/Cinnamon combination problem: for me the problem appears 100% of the times launching a video with intel activated, 0% of the times if I switch to NVidia and relog on the desktop.

**Comment by rebelwocause** [comment]
Same Problem:

Kernel: 4.4.0-21-generic x86_64 (64 bit)
Desktop: Cinnamon 3.2.7
Distro: Linux Mint 18.1 Serena
CPU:  Quad core Intel Core i5-4460 (-MCP-) cache: 6144 KB
Graphics Card: Intel Xeon E3-1200 v3/4th Gen Core Processor Integrated Graphics Controller
Display Server: X.Org 1.18.3 drivers: intel (unloaded: fbdev,vesa)
Resolution: 1920x1080 @ 60.00hz
GLX Renderer: Mesa DRI Intel Haswell Desktop GLX Version: 3.0 Mesa 11.
